group_vars/vagrant

```diff
@@ -5,8 +5,8 @@ consul_advertise: "{{ ansible_eth1.ipv4.address }}"
 consul_bind_addr: "{{ ansible_eth1.ipv4.address }}"
 marathon_hostname: "{{ ansible_eth1.ipv4.address }}"
 mesos_hostname: "{{ ansible_eth1.ipv4.address }}"
-mesos_master_quorum: 2
```

Comment:
This (mesos_master_quorum and consul_bootstrap_expect) could probably be removed from here now as we are  passing it explicitly.